packages/core/src/common/keybinding.ts

```diff
@@ -26,6 +26,13 @@ export interface Keybinding {
     readonly accelerator?: Accelerator;
 }
 
+export interface RawKeybinding {
+    command: string;
+    keybinding: string;
+    context?: string;
+    args?: string[];
```

Comment:
what is args?

Reply:
default args for the command, the same as VS code

Reply:
I think I should remove that, looking at vscode their custom keybindings structure is simply { command, keybinding and when (context) }. Maybe we should go with a similar approach.